.travis.yml

```diff
@@ -49,6 +49,9 @@ before_install:
 install:
   - travis_retry composer install --no-interaction --ignore-platform-reqs
 
+before_script:
+  echo 'dmbstring.func_overload=7' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
```

Comment:
If we set mbstring.func_overload=7 we force the overload of strlen() to mb_strlen() and substr to mb_substr(). Because we are missing the $encoding optional parameter in mb_ functions this will fail, using the UTF-8 encoding (4 bytes for 1 character). Basically, this is a trick to check for substr() and strlen() usage in the code.